src/binding.cc

```diff
@@ -995,156 +972,120 @@ class Client : public ObjectWrap {
-      cfg_ssl_ca_symbol = NODE_PSYMBOL("ca");
-      cfg_ssl_capath_symbol = NODE_PSYMBOL("capath");
-      cfg_ssl_cipher_symbol = NODE_PSYMBOL("cipher");
-      cfg_ssl_reject_symbol = NODE_PSYMBOL("rejectUnauthorized");
-      cfg_local_infile_symbol = NODE_PSYMBOL("local_infile");
-      cfg_read_default_file = NODE_PSYMBOL("read_default_file");
-      cfg_read_default_group = NODE_PSYMBOL("read_default_group");
-      cfg_charset_symbol = NODE_PSYMBOL("charset");
-
-      target->Set(name, Client_constructor->GetFunction());
+      NanScope();
+
+      Local<String> name = NanNew<String>("Client");
+
+      Local<FunctionTemplate> tpl = NanNew<FunctionTemplate>(New);
+      tpl->InstanceTemplate()->SetInternalFieldCount(1);
+      tpl->SetClassName(name);
+
+      NanAssignPersistent(Client_constructor, tpl);
+
+      NODE_SET_PROTOTYPE_METHOD(tpl, "connect", Connect);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "query", Query);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "abortQuery", AbortQuery);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "escape", Escape);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "end", Close);
+      NODE_SET_PROTOTYPE_METHOD(tpl, "isMariaDB", IsMariaDB);
+
+      // Make it visible in JavaScript land
+      target->Set(name, tpl->GetFunction());
     }
 };
 
-static Handle<Value> Escape(const Arguments& args) {
-  HandleScope scope;
+static NAN_METHOD(Escape) {
+  NanScope();
 
   if (args.Length() == 0 || !args[0]->IsString()) {
-    return ThrowException(Exception::Error(
-      String::New("You must supply a string"))
-    );
+    return NanThrowError("You must supply a string");
   }
+
   String::Utf8Value arg_v(args[0]);
   unsigned long arg_len = arg_v.length();
   char *result = (char*) malloc(arg_len * 2 + 1);
-  unsigned long result_len =
-          mysql_escape_string_ex(result, (char*)*arg_v, arg_len, "utf8");
-  Local<String> escaped_s = String::New(result, result_len);
+  unsigned long result_len = mysql_escape_string_ex(result, (char*)*arg_v, arg_len, "utf8");
+  Local<String> escaped_s = NanNew<String>(result, result_len);
   free(result);
-  return scope.Close(escaped_s);
+
+  NanReturnValue(escaped_s);
 }
 
-static Handle<Value> Version(const Arguments& args) {
-  HandleScope scope;
-  return scope.Close(String::New(mysql_get_client_info()));
+static NAN_METHOD(Version) {
+  NanScope();
+
+  NanReturnValue(NanNew<String>(mysql_get_client_info()));
 }
 
 extern "C" {
   void init(Handle<Object> target) {
-    HandleScope scope;
+    NanScope();
+
     Client::Initialize(target);
-    target->Set(String::NewSymbol("escape"),
-                FunctionTemplate::New(Escape)->GetFunction());
-    target->Set(String::NewSymbol("version"),
-                FunctionTemplate::New(Version)->GetFunction());
+    target->Set(NanNew<String>("escape"),   NanNew<FunctionTemplate>(Escape)->GetFunction());
+    target->Set(NanNew<String>("version"),  NanNew<FunctionTemplate>(Version)->GetFunction());
```

Comment:
These were previously wrapped to new lines because the lines exceed 80 characters.